ironfish/src/account/accounts.ts

```diff
@@ -957,6 +963,8 @@ export class Accounts {
 
     this.accounts.delete(name)
     await this.db.removeAccount(name)
+
+    await this.onAccountRemoved.emitAsync(name)
```

Comment:
Do we need async properties for these events? Should the accounts need to wait for the receiver to finish the event callback?